docs/dev/connectors/streamfile_sink.md

```diff
@@ -110,12 +110,45 @@ interactions of bucket assigners and rolling policies.
 
 In the above example we used an `Encoder` that can encode or serialize each
 record individually. The streaming file sink also supports bulk-encoded output
-formats such as [Apache Parquet](http://parquet.apache.org). To use these,
-instead of `StreamingFileSink.forRowFormat()` you would use
+formats such as [Apache Parquet](http://parquet.apache.org) or [Apache Avro](https://avro.apache.org/).
+To use these, instead of `StreamingFileSink.forRowFormat()` you would use
 `StreamingFileSink.forBulkFormat()` and specify a `BulkWriter.Factory`.
 
-[ParquetAvroWriters]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/formats/parquet/avro/ParquetAvroWriters.html)
-has static methods for creating a `BulkWriter.Factory` for various types.
+<div class="codetabs" markdown="1">
+<div data-lang="java" markdown="1">
+{% highlight java %}
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.formats.parquet.avro.ParquetAvroWriters;
+import org.apache.flink.streaming.api.functions.sink.filesystem.StreamingFileSink;
+
+DataStream<RawEvent> input = ...;
+
+StreamingFileSink<RawEvent> sink = StreamingFileSink.forBulkFormat(
+				new Path(outputPath),
+				ParquetAvroWriters.forSpecificRecord(RawEvent.class)
+)
+
+input.addSink(sink);
+
+{% endhighlight %}
+</div>
+<div data-lang="scala" markdown="1">
+{% highlight scala %}
+import org.apache.flink.core.fs.Path;
+import org.apache.flink.formats.parquet.avro.ParquetAvroWriters
+import org.apache.flink.streaming.api.functions.sink.filesystem.StreamingFileSink
+
+val input: DataStream[RawEvent] = ...
+
+val sink: StreamingFileSink[RawEvent] = StreamingFileSink
+    .forRowFormat(new Path(outputPath), ParquetAvroWriters.forSpecificRecord(RawEvent.class))
```

Comment:
This should probably be bulk format

Reply:
Thanks, copy paste. I had the code in Java, so I had to construct the Scala example by hand :-)